plugins/inputs/prometheus_scraper/metrics_receiver_test.go

```diff
@@ -1,112 +1,172 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: MIT
+// Copyright The OpenTelemetry Authors
```

Comment:
So this code is taken directly from the OTel receiver?

Reply:
Its part of copy paste for developing unit test. I learn some of their unit test and Prometheus unit test before able to write down this unit test so it is not taken directly from the OTEL receiver.